utils/commands/misc.lua

```diff
@@ -499,6 +499,10 @@ function Public.reset()
     this.players = {}
 end
 
+function Public.set_always_allow_clear_corpses(val)
```

Comment:
This can be removed.

Reply:
resolved in 26f34ea8d5d108a6cbd20848b24bc02525c5ff98